bot.py

```diff
@@ -44,9 +46,36 @@ async def on_message(msg):
     if msg.channel.name == CHANNEL and len(msg.attachments) > 0:
         #read the msg, put it into a stream for consumption
         try:
+            # Look for user submitted deck record from the message.
+            record_regex = re.compile(r'\b[0-3]-[0-3]\b')
```

Comment:
I think it would be a little more extensible if you used a capture group here to obtain the number of wins:
```

record_regex = re.compile(r'\b([0-3])-[0-3]\b')
record_match = record_regex.match(msg.content)
if record_match:
    wins = record_match[1]
```